samples/highcharts/chartchooser/categorical-distribution-bar-pattern/demo.js

```diff
@@ -0,0 +1,120 @@
+// Age categories
+const categories = [
+    "0-14",
+    "15-24",
+    "25-44",
+    "45-64",
+    "65-84",
+    "85+",
+    "30-34",
+    "35-39",
+    "40-44",
+    "45-49",
+    "50-54",
+    "55-59",
+    "60-64",
+    "65-69",
+    "70-74",
+    "75-79",
+    "80-84",
+    "85-89",
+    "90-94",
+    "95-99",
+    "100 + "
+];
+const maleColor = { patternIndex: 0 },
+    femaleColor = { patternIndex: 5 };
+Highcharts.chart("container", {
+    chart: {
+        type: "bar"
+    },
+    title: {
+        text: "Denmark Population Pyramid 2019"
+    },
+    subtitle: {
+        text:
+      'Source: <a href="https://www.populationpyramid.net/denmark/2019/">populationpyramid</a>'
+    },
+    accessibility: {
+        point: {
+            valueDescriptionFormat: "{index}. Age {xDescription}, {value}%."
+        }
+    },
+    xAxis: [
+        {
+            categories: categories,
+            reversed: false,
+            labels: {
+                step: 1
+            },
+            accessibility: {
+                description: "Age (male)"
+            }
+        },
+        {
+            // mirror axis on right side
+            opposite: true,
+            reversed: false,
+            categories: categories,
+            linkedTo: 0,
+            labels: {
+                step: 1
+            },
+            accessibility: {
+                description: "Age (female)"
+            }
+        }
+    ],
+    yAxis: {
+        title: {
+            text: null
+        },
+        labels: {
+            formatter: function () {
+                return Math.abs(this.value) + "%";
+            }
+        },
+        accessibility: {
+            description: "Percentage population",
+            rangeDescription: "Range: 0 to 5%"
+        }
+    },
```

Comment:
A better way to display a population pyramid is to use two separate y-axes, one reversed and one not. Then we avoid the hack of setting negative data values and subsequent hacking the label and tooltip formatters. See https://jsfiddle.net/highcharts/n60wLcy1/.

Reply:
Done 👍